umh.docs.umh.app/content/en/docs/features/datainfrastructure/network-view.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Network Monitoring
+menuTitle: Network Monitoring
+description: |
+  Monitor all instances within the Unified Namespace at a glance.
+weight: 2000
+draft: true
+---
+
+The Network Monitoring feature of the United Manufacturing Hub visualizes the instance's
+network within the Unified Namespace.
+
+## When should I use it?
+
+Your infrastructure might consist of multiple UMH instances. This feature offers
+an intuitive view for monitoring the entire network of all instances and their
```

Comment:
it is not exactly about monitoring the network. It is about monitoring the connected netwrok devices and external services.

This is required for extracting data from a device via a protocol converter, as it will help in the troubleshooting if something goes wrong (e.g., if the device is offline, then it will tell you that, so that you don't need to look into the protocol converter first)